src/Symfony/Component/Form/FormInterface.php

```diff
@@ -103,6 +103,15 @@ public function all();
      */
     public function getErrors($deep = false, $flatten = true);
 
+    /**
+     * Removes all the errors of this form.
+     *
+     * @param bool $deep Whether to remove errors of child forms as well
+     *
+     * @return FormInterface The form instance
+     */
+    public function clearErrors($deep = false);
```

Comment:
I don't think this will be possible. You'll have to make this work for 4.x as well and as is, it's a BC break. Probably requires a new interface, which can be merged in 5.0.

Reply:
Do you mean I should create something like `FormClearErrorsInterface` and make `Form` implement that?

If we want to merge it into `FormInterface` in 5.0, does that mean the `FormClearErrorsInterface` should be marked as deprecated right away, so that people will be aware that is goes away in version 5.0?

I have not done something like this before, so that is why I would apreciate some guideance in how to do it. Thanks.

Reply:
Also @dosten wrote in https://github.com/symfony/symfony/pull/14233#issuecomment-90086837 that the BC break would not be a problem.

Reply:
> Also @dosten wrote in #14233 (comment) that the BC break would not be a problem.

The BC promise states that adding a new method is not acceptable http://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces

Reply:
@TerjeBr The comment you linked dates from 2015, since then the concept of `@api` interface was removed and we now consider every class as part of the public API (https://github.com/symfony/symfony-docs/pull/5735). This means we can't add methods to any interface.